charts/eks/templates/daemonset-hostpath-mapper.yaml

```
@@ -0,0 +1,99 @@
+{{- if .Values.hostpathMapper.enabled }}
+apiVersion: apps/v1
+{{- if not .Values.hostpathMapper.dev }}
+kind: DaemonSet
+{{- else  }}
+kind: Deployment
+{{- end }}
+metadata:
+  name: {{ .Release.Name }}-hostpath-mapper
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: vcluster-hostpath-mapper
+    component: hostpath-mapper
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+  {{- if .Values.annotations }}
+  annotations:
+{{ toYaml .Values.annotations | indent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.hostpathMapper.dev }}
+  replicas: 1
+  {{- end }}
+  selector:
+    matchLabels:
+      app: vcluster-hostpath-mapper
+      release: {{ .Release.Name }}
+      component: hostpath-mapper
+  template:
+    metadata:
+      labels:
+        app: vcluster-hostpath-mapper
+        release: {{ .Release.Name }}
+        component: hostpath-mapper
+    spec:
+      {{- if .Values.serviceAccount.name }}
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      {{- else }}
+      serviceAccountName: vc-{{ .Release.Name }}
+      {{- end }}
+      containers:
+      - name: hostpath-mapper
+        {{- if .Values.hostpathMapper.image }}
+        image: "{{ .Values.defaultImageRegistry }}{{ .Values.hostpathMapper.image }}"
+        {{- else }}
+        image: "{{ .Values.defaultImageRegistry }}loftsh/vcluster:{{ .Chart.Version }}"
+        {{- end }}
+        {{- if .Values.syncer.workingDir }}
+        workingDir: {{ .Values.syncer.workingDir }}
+        {{- end }}
+        command:
+          - /vcluster
+        args:
+          - maplogs
```

```suggestion
          - maphostpaths
```
This needs to be fixed for all distros